cli/azd/pkg/infra/provisioning/bicep/bicep_provider.go

```diff
@@ -95,6 +95,52 @@ func (p *BicepProvider) EnsureConfigured(ctx context.Context) error {
 	return p.prompters.EnsureSubscriptionLocation(ctx, p.env)
 }
 
+func (p *BicepProvider) promptResourceGroupName(ctx context.Context) (string, error) {
+	// Get current resource groups
+	groups, err := p.azCli.ListResourceGroup(ctx, p.env.GetSubscriptionId(), nil)
+	if err != nil {
+		return "", fmt.Errorf("listing resource groups: %w", err)
+	}
+
+	choices := make([]string, len(groups)+1)
+	choices[0] = "Create a new resource group"
+	for idx, group := range groups {
+		choices[idx+1] = fmt.Sprintf("%d. %s", idx+1, group.Name)
```

Comment:
Good call - I had actually thought I added this code but it must have gotten reverted.  I'll do a sort by ascending order here and update the test mock so that the data is returned in an unsorted order.